### ISSUE TITLE
Printer Plugin

### ISSUE THREAD

**Issue by BarisCanYesil** [capacitor-community/proposals#214]
## Plugin Request

**Name**: Printer Plugin
**Package**: `@capacitor-community/printer`

### Platform(s)
Android 5.1+
iOS 13+

### Supported Contents
HTML (can use CSS inline styling)
Text
Image(as HTML)
Base64 (img, PDF)

### Existing Solutions
Capacitor v5.x & Capacitor v6.x
https://github.com/BarisCanYesil/capacitor-plugin-printer

### Description

Capacitor plugin for printing HTML format value iOS/Android apps.

I use the plugin I wrote myself.
I would like to add it under "@capacitor-community/printer"

**Comment by shiv19** [comment]
@BarisCanYesil 
https://capawesome.io/plugins/printer/

**Comment by ipehimanshu** [comment]
yes true, it will be best

**Comment by StriderRanger** [comment]
There is no official capacitor plugin for printing. I tested in capacitor 6 the old cordova-plugin-printer and its wrapper @awesome-cordova-plugins/printer but they are not working, the gradle build is failing.
There must be a plugin that supports printing in capacitor.

**Comment by robingenz** [comment]
> There must be a plugin that supports printing in capacitor.

Yes, there is one:: https://capawesome.io/plugins/printer/

**Comment by StriderRanger** [comment]
I searched for @capawesome-team/capacitor-printer on npm, and it's not available. Since it’s Sponsorware, does that mean we have to wait for it to be available for open-source use?

**Comment by robingenz** [comment]
Yes, this plugin is currently only available as [sponsorware](https://capawesome.io/sponsors/insiders/).

**Comment by h7nif** [comment]
Voting for this. Should be made official
